src/actions/helpers/sparkpostApiError.js

```diff
@@ -4,6 +4,9 @@ import _ from 'lodash';
 function SparkpostApiError(error) {
   const apiError = _.get(error, 'response.data.errors[0]', {});
 
+  if (!error.response) {
+    error.message = 'You may be having network issues or an adblocker may be blocking part of the app.';
+  }
   this.name = 'SparkpostApiError';
   this.message = apiError.description || apiError.message || error.message;
```

Comment:
You could move the conditional here. Something like 
```suggestion
   this.message = apiError.description || apiError.message ||
    (error.response ? error.message : 'You may be having network issues or an adblocker may be blocking part of the app.');
```

Reply:
Doesn't really make too much of a difference. Up to you.

Reply:
I prefer your way since it's better than just overwriting errorMessage